admin_guide/managing_networking.adoc

```diff
@@ -144,6 +144,36 @@ $ systemctl restart atomic-openshift-master-api atomic-openshift-master-controll
 
 ----
 
+[[admin-guide-managing-networking-ingress]]
+== Setting up the HAProxy router to accept ingress objects
+
+By default, the HAProxy router only accepts route objects. You can set up the
+HAProxy router as a Kubernetes
+link:https://kubernetes.io/docs/concepts/services-networking/ingress/[Ingress
+Controller] so that it accepts both route and ingress objects. If configured,
+the same rules apply for claiming a domain for a namespace based on the creation
+time of the object.
```

Comment:
I'm not sure what the above means. Can this be elaborated on? Is this listed somewhere else in the docs (it mentions "same rules")?

Reply:
Hm... the best I can find is a tangential reference in install_config/router/default_haproxy_router.adoc at arond line 1218.